eclipse/openvsx
# Update extension preview when publishing a new version.

Fixes #423 

Testing steps:

1. publish [vscode-yaml 0.23.0](https://github.com/redhat-developer/vscode-yaml/releases/download/0.23.0/yaml-0.23.0-19445.vsix)
2. execute query: `SELECT e.name, ev.version, e.preview, ev.pre_release FROM extension e JOIN extension_version ev ON ev.extension_id = e.id WHERE e.name = 'vscode-yaml';` `preview` should be true, `pre_release` should be false.
3. Run git clone https://github.com/redhat-developer/vscode-yaml.git
4. Update package.json with version "version": "1.7.0" and engine "vscode": "^1.63.0"
5. Run in vscode-yaml folder:
`yarn`
`export OVSX_REGISTRY_URL=http://localhost:8080`
`export OVSX_PAT=super_token`
`ovsx publish --pre-release`
6. Repeat step 2, `preview` should now be false for both extension versions. `pre_release` should be true for version 1.7.0 and false for version 0.23.0
